models/zendesk__ticket_metrics.sql

```diff
@@ -53,8 +53,14 @@ with ticket_enriched as (
 
 select
   ticket_enriched.*,
-  ticket_reply_times_calendar.first_reply_time_calendar_minutes,
-  ticket_reply_times_calendar.total_reply_time_calendar_minutes,
+  case when coalesce(ticket_comments.count_public_agent_comments, 0) = 0 and ticket_enriched.status = 'solved'
```

Comment:
tickets that are not solved and have 0 comments would still have a really high `first_reply_time_calendar_minutes` -- why should this depend on the resolution status?

Reply:
Good catch it should not depend on resolution status!

Reply:
Removed